synapse/tests/test_lib_storm.py

```diff
@@ -1418,6 +1418,42 @@ def f2(mesg):
                 self.eq(node[1].get('tufo:form'), 'task')
                 self.isin(node[1].get('task'), ('hehe:haha', 'wow'))
 
+    def test_storm_vartree(self):
+
+        quer = s_storm.Query()
+
+        vals = ['x', 'y']
+        quer.addTreeVar('a', vals)
+
+        self.len(0, list(quer.iterVarTree('asdf')))
+
+        for node, varz in quer.iterVarTree('a'):
+            quer.addTreeVar('b', ['z','q'], tree=node)
+
+        rets = []
+        for node, varz in quer.iterVarTree('a.b'):
+            rets.append(tuple(sorted(varz.items())))
+
+        rets.sort()
+        self.eq(rets, [(('a', 'x'), ('a.b', 'q')), (('a', 'x'), ('a.b', 'z')), (('a', 'y'), ('a.b', 'q')), (('a', 'y'), ('a.b', 'z'))])
+
+        with self.getRamCore() as core:
+            core.ask('[ inet:ipv4=1.2.3.4 :cc=us inet:dns:a=vertex.link/1.2.3.4 ]')
+            core.ask('$dns={inet:dns:a:fqdn=woot.com} $dns.ipv4={ :ipv4->inet:ipv4 }')
```

Comment:
This test fails for me

Reply:
This test fails because `$dns.ipv4` contains a number and numbers are not in `setset`.

Reply:
The first character should not be a digit